app/views/pages/about.html.erb

```diff
@@ -7,16 +7,32 @@
     <div>
       <h3 class="font-bold text-2xl mt-4 mb-2">About</h3>
       <p>
-        The Registered Apprenticeship Standards Library is part of the Urban Institute’s
-        <%= link_to "Registered Apprenticeship Occupations and Standards Center of Excellence.", "https://www.urban.org/policy-centers/center-labor-human-services-and-population/projects/registered-apprenticeship-occupations-and-standards-center-excellence", class: "font-semibold hover:underline" %>
-        In partnership with the U.S. Department of Labor’s Office of Apprenticeship,
-        the mission of the Center of Excellence is to build a gold-standard
-        occupational skills infrastructure that increases the transparency,
-        portability, quality, usability, and scalability of the US apprenticeship
-        system. In addition to this standard's library, the Center of Excellence
-        is developing more than 80 National Occupational Frameworks,
-        institutionalizing best practices, and promoting methodologies to ensure
-        a sustainable, high-quality registered apprenticeship system.
+        The Registered Apprenticeship Standards Library is a central repository
+        of approved apprenticeship programs from across the country. Whether you
+        are just beginning to explore apprenticeships or you are ready to
+        develop or update a program, the Library can help you easily find
+        examples of approved programs. The Library is a comprehensive collection
+        of apprenticeship program training standards registered at both the
+        national and state levels and includes standards for hundreds of
+        occupations across all industry sectors.
       </p>
+
+      <h4 class="font-bold text-xl mt-4 mb-2">How to Use the Registered Apprenticeship Standards Library</h4>
+      <p class="mb-1">The Library can help you:</p>
+      <ul class="list-disc list-inside" style="list-style-type: disc;">
+        <li class="mb-1.5"><strong><em>Search by industry, occupation, state, and more</em></strong> to find the apprenticeship program training standards most helpful for you.</li>
+        <li class="mb-1.5"><strong><em>Find multiple examples of program standards for one occupation</em></strong> to provide options and ideas for your own apprenticeship.</li>
+        <li class="mb-1.5"><strong><em>Access approved work process schedules and related instruction outlines</em></strong> you can use as a starting point.</li>
+        <li><strong><em>Build from existing program standards</em></strong> to help you more quickly design your apprenticeship program.</li>
+      </ul>
+
+      <h4 class="font-bold text-xl mt-6 mb-2">Additional Resources</h4>
+      <ul class="list-disc list-inside" style="list-style-type: disc;">
```

Comment:
@jenlu33 @FerPerales  (@fatimaburke ?) I cannot get the tailwind `list-disc` to work (I hard-coded a `style` here). `list-decimal` works without issue.